PhotoPlusFrontend/PhotoPlus/src/app/app.module.ts

```diff
@@ -1,125 +1,130 @@
-    TopicRemoveComponent,
-    PostComponent,
-    ProductComponent,
-    OrderComponent,
-    AdminPanelComponent,
-    ManageMultipleOrdersComponent,
-    ManageProductComponent,
-    DeleteUsersComponent,
-    GenerateReportComponent,
-    ChangePrivilegesComponent,
-    AddDeliveryComponent,
-    ManageSingleOrderComponent,
-    ImageCarouselComponent,
-    ImageDisplayComponent,
-    PageSelectComponent,
-    AboutComponent
-  ],
-  imports: [
-    BrowserModule,
-    BrowserAnimationsModule,
-    AppRoutingModule,
-    ReactiveFormsModule,
-    HttpClientModule,
-    MatCardModule,
-    MatDividerModule,
-    MatExpansionModule,
-    MatPaginatorModule,
-    NgbModule,
-    FormsModule,
-    MatSelectModule
-  ],
-  providers: [
-    {
-      provide: HTTP_INTERCEPTORS,
-      useClass: AuthInterceptor,
-      multi: true
-    },
-    {
-      provide: HTTP_INTERCEPTORS,
-      useClass: ErrorInterceptor,
-      multi: true
-    },
-    {
-      provide: MatPaginatorIntl,
-      useClass: CustomMatPaginatorIntl
-    }
-  ],
-  bootstrap: [AppComponent]
+    declarations: [
+        AppComponent,
+        HomeComponent,
+        ProductsComponent,
+        RegistrationComponent,
+        LoginComponent,
+        CartComponent,
+        ErrorModalComponent,
+        SuccessModalComponent,
+        CategoryComponent,
+        ForumComponent,
+        SectionComponent,
+        SectionBodyComponent,
+        SectionEditComponent,
+        SectionRemoveComponent,
+        SectionAddComponent,
+        TopicBodyComponent,
+        TopicHeaderComponent,
+        TopicAddComponent,
+        TopicEditComponent,
+        TopicRemoveComponent,
+        PostComponent,
+        ProductComponent,
+        OrderComponent,
+        AdminPanelComponent,
+        ManageMultipleOrdersComponent,
+        ManageProductComponent,
+        DeleteUsersComponent,
+        GenerateReportComponent,
+        ChangePrivilegesComponent,
+        AddDeliveryComponent,
+        ManageSingleOrderComponent,
+        ImageCarouselComponent,
+        ImageDisplayComponent,
+        PageSelectComponent,
+        AboutComponent,
+        UserDetailsComponent,
+        UserDataComponent,
+        UserAddressComponent
+    ],
+    imports: [
+        BrowserModule,
+        BrowserAnimationsModule,
+        AppRoutingModule,
+        ReactiveFormsModule,
+        HttpClientModule,
+        MatCardModule,
+        MatDividerModule,
+        MatExpansionModule,
+        MatPaginatorModule,
+        NgbModule,
+        FormsModule,
+        MatSelectModule
+    ],
+    providers: [
+        {
+            provide: HTTP_INTERCEPTORS,
+            useClass: AuthInterceptor,
+            multi: true
+        },
+        {
+            provide: HTTP_INTERCEPTORS,
+            useClass: ErrorInterceptor,
+            multi: true
+        },
+        {
+            provide: MatPaginatorIntl,
+            useClass: CustomMatPaginatorIntl
+        }
+    ],
+    bootstrap: [AppComponent]
```

Comment:
wrong formatting

Reply:
As below we do not agreed for one formatting style. I use editor with different formatting. So do not expect that I will use yours favourite because this formatting is wrong according to You.

Reply:
We did. We did when everybody who worked on frontend used "our favourite" style. 
I do not expect you to use our formatting because this one is wrong according to me, I do expect you to use our formatting because it has settled in as the one for this project.
Besides, this file wasnt written by you, it was for some reason only edited just to change formatting.

Reply:
Why was this conversation marked as resolved? Havent seen any commits resolving problem I described